NeosModLoader/NeosModLoader.csproj

```diff
@@ -13,7 +13,7 @@
 		<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
 		<TargetFramework>net462</TargetFramework>
 		<FileAlignment>512</FileAlignment>
-		<LangVersion>9.0</LangVersion>
+		<LangVersion>latestMajor</LangVersion>
```

Comment:
... I'm unsure if it's a good idea for NML to not define a concrete version number, as I've seen "latest" default to different numbers on different system (like for example an outdated CI system)

Reply:
Wouldn't it fail to build just the same if it had a concrete version number higher than what the CI system supports?

Reply:
I think the idea is that we'd rather not get sideswiped by M$ pushing a new LangVersion that breaks source compatibility and therefore breaks our build randomly. If you need features from a LangVersion > 9.0 I'd rather you just bump the version here than set it to something dynamic.

Reply:
Right, that's true indeed